packages/framework/collabspaces/src/collabSpaces.ts

```diff
@@ -569,9 +619,46 @@ export class CollabSpacesRuntime
 	public async getCellDebugInfo(
 		row: number,
 		col: number,
-	): Promise<{ channel?: ICollabChannelCore }> {
+	): Promise<{
+		channel?: ICollabChannelCore;
+		channelId?: string;
+		rowId?: string;
+		colId?: string;
+	}> {
 		const result = this.getCellInfo(row, col);
-		return { channel: await result.channel };
+		const { rowId = undefined, colId = undefined } =
+			result.channelId !== undefined ? this.parseChannelId(result.channelId) : {};
+		const channel = await result.channel;
+		return { channel, channelId: result.channelId, rowId, colId };
```

Comment:
I wonder if you can do the following:

```ts
const info = result.channelId !== undefined ? this.parseChannelId(result.channelId);
const channel = await result.channel;
return { channel, ...info};
```
it returns a bit more info (iteration), but that should not matter. Feels a bit more readable. 
(this assumes that tsc is happy with it)

Reply:
I do not have  a strong opinion either way  - I will change it.